src/Appwrite/Platform/Modules/Functions/Workers/Builds.php

```diff
@@ -1144,7 +1130,7 @@ protected function buildDeployment(
                 $branchName = $deployment->getAttribute('providerBranch');
                 if (!empty($branchName)) {
                     $sitesDomain = System::getEnv('_APP_DOMAIN_SITES', '');
-                    $domain = "branch-{$branchName}-{$resource->getId()}-{$project->getId()}.{$sitesDomain}";
+                    $domain = "branch-{$branchName}.{$sitesDomain}";
```

Comment:
Irellevant to PR title, but fixes problem Khushboo is facing with too long DNS name. It was fixed in another place, but not here.